package day_3.exceptions;

import day_2.Test;

public class MainException {
    public static void main(String[] args) {
        System.out.println("a/b" + Test.devide(5, 0));
    }
}
